src/core/Table/TableRowMemoized.tsx

```diff
@@ -55,7 +55,7 @@ const TableRow = <T extends Record<string, unknown>>(props: {
 
   const userRowProps = rowProps?.(row);
   const mergedProps = {
-    ...row.getRowProps(),
+    ...row.getRowProps({ style: { flex: `0 0 auto` } }),
```

Comment:
Body now has `flex-direction: column` and `react-table` sets row style to `flex: 1 0 auto` which makes row grow in height when table has `height: 100%`:
![image](https://user-images.githubusercontent.com/36186912/137865593-55d742e6-e39c-4f34-9256-b8a008f59389.png)